www/pages/user/[id]/home.js

```diff
@@ -12,25 +12,32 @@ import styles from "../../../styles/pages/user/[id]/home.module.css";
 
 import { notify } from "../../../store/features/snackbar";
 
-const User = (props) => {
-  const [loading, setLoading] = useState(true);
-  const [ownedMeetings, setOwnedMeetings] = useState([]);
-  const [participantMeetings, setParticMeetings] = useState([]);
+const User = ( props ) => {
+
+  const filtersConstuctor = { owners: [], name: '' };
```

Comment:
What do you think about calling this `initialFilters` or something like that. `filtersConstructor` implies that it is a constructor function.